src/main/java/net/minecraftforge/fmp/microblock/IMicroblockContainerTile.java

```diff
@@ -0,0 +1,34 @@
+package net.minecraftforge.fmp.microblock;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.util.IWorldLocation;
+import net.minecraftforge.fmp.multipart.IMultipartContainer;
+
+/**
+ * Interface that represents a {@link TileEntity} which can contain microblocks (but not other multiparts). You can add
+ * this to your {@link TileEntity} using {@link Capability Capabilities}.
+ *
+ * @see IMicroblock
+ * @see IMultipartContainer
+ */
+public interface IMicroblockContainerTile extends IWorldLocation
```

Comment:
IMicroblockContainerTile also allows you to decide if you want a microblock to be added or not without implementing your own IMicroblockContainer and has a very easy way of notifying the tile when one of the microblocks in it changes.